pkg/machines/components/vcpuModal.jsx

```diff
@@ -157,15 +157,18 @@ export class VCPUModal extends React.Component {
         const { vm } = this.props;
         let caution = null;
 
-        if (vm.state === 'running') {
+        if (vm.state === 'running' && (
+            this.state.sockets != (vm.cpu.topology.sockets || 1) ||
+            this.state.threads != (vm.cpu.topology.threads || 1) ||
+            this.state.cores != (vm.cpu.topology.cores || 1) ||
+            this.state.max != vm.vcpus.max ||
+            this.state.count != vm.vcpus.count)
+        ) {
```

Comment:
The commit message suggests that this is a visual/design change; i. e. replacing the Alert with an `idle-message`. How  does that look like now, I suppose like in issue #10526?

This code change here changes the behaviour though, not the design. Do I understand this correctly? Previously it would also show the banner if the values were changed, and then manually re-changed back to what the current libvirt VM values are. Now the warning gets removed again as soon as that happens. The commit message should explain this.

Reply:
Correct, and the screenshots are here:

VCPU warning old             |  VCPU warning new
:-------------------------:|:-------------------------:
![vcpu-footer-old](https://user-images.githubusercontent.com/14921356/48413563-5f758900-e748-11e8-900b-249d1b9e5833.png)  |  ![vcpu-footer-warning](https://user-images.githubusercontent.com/14921356/48413421-fe4db580-e747-11e8-810c-e3830f4bc3b9.png)

I 'll adjust the commit message.